common/src/main/java/com/bloxbean/cardano/client/util/IntUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.bloxbean.cardano.client.util;
+
+public class IntUtil {
```

Comment:
There is a Utils class which has a method with similar functionality ``uint32ToByteArrayBE``. But it's currently in crypto/bip32 package.

I am just wondering if we can use that. Otherwise, we can keep this class but rename it to ByteUtil, allowing us to add additional conversion methods, such as long to bytes, etc later.

Reply:
So yes the function would work as well. For me this conversions seemed pretty common so I wasn't looking into the bip32 package for this functionality. 
I renamed the package. We can discuss if moving the other functions to a more common place.